pkg/registry/porch/packagecommon.go

```diff
@@ -210,9 +218,21 @@ func (r *packageCommon) updatePackageRevision(ctx context.Context, name string,
 		return nil, false, apierrors.NewBadRequest("namespace must be specified")
 	}
 
+	pkgMutexKey := fmt.Sprintf("%s/%s", ns, name)
```

Comment:
Seems to be some degree of duplication across the package**.go.
Would it be worth trying to extract a function for this?

Reply:
True - extracting will let us keep the format unified, anyway. Done, though I had to create a separate one for the Create() operation.